frontend/public/config.js

```diff
@@ -1,10 +1,6 @@
 // This file is provided as an option to support runtime loading of configuration parameters in OpenShift
 // Development builds and production builds using yarn build will still work using .env files
-const config = (() => {
-  return {
-    // VITE_BACKEND_URL: '',
-    // VITE_KEYCLOAK_URL: '',
-    // VITE_KEYCLOAK_CLIENT_ID: '',
-    // VITE_KEYCLOAK_REALM: ''
-  };
-})();
+
+// window.localStorage.setItem("VITE_KEYCLOAK_URL", "");
+// window.localStorage.setItem("VITE_KEYCLOAK_CLIENT_ID", "");
+// window.localStorage.setItem("VITE_NODE_ENV", "");
```

Comment:
set window local storage to store runtime env vars, so won't have the error complains config is undefined when run tests